viewer/packages/utilities/src/computeSceneBoundingBox.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { SectorMetadata } from '@reveal/cad-parsers';
```

Comment:
Utilities should not have a dependency to cad-parsers. Consume a list of bounds in `computeSceneBoundingBox` instead.